Source/spells.cpp

```diff
@@ -158,7 +158,7 @@ void EnsureValidReadiedSpell(PlayerStruct &player)
 	}
 }
 
-bool CheckSpell(int id, int sn, char st, bool manaonly)
+bool CheckSpell(int id, spell_id sn, spell_type st, bool manaonly)

```

Comment:
_Huge_ improvement to readability here.

Reply:
Yeah was really happy to find that this wasnt left out because of relation issues.